contracts/optimistic-ethereum/libraries/rlp/Lib_RLPWriter.sol

```diff
@@ -89,16 +89,7 @@ library Lib_RLPWriter {
             bytes memory _out
         )
     {
-        _malloc(0x20);
```

Comment:
I think that, with this change, `_malloc` is no longer used -- can you check and delete if true?